### PR TITLE
Added not found page

### LINKED ISSUE #324: Create a 404 Page
## Points to remember
- For reference - https://members.realdevsquad.com/x
- Navbar should be there navigate across the pages
- Footer can also be there.
- For visiting every route of `https://my.realdevsquad.com/` which doesn't exist redirect to this 404 page
- Please direct the user to this 404 route here
https://github.com/Real-Dev-Squad/website-my/blob/327dc1588709cb48a52974be3f87a5bf273b79de/app/routes/notifications.js#L15

## Image to Use
page-not-found

### PR DESCRIPTION
**Pull request by satyam73**
Closes #324

I have added not found page for the issue #324 . Please check it and tell me if there is something which needs to be updated.

Image Reference :
image

**Updated Image**

image

> Desktop Devices

image

> Tablet Devices

image

> Mobile Devices

### CODE REVIEW

File: app/router.js [hunk 1]
```diff
@@ -13,4 +13,5 @@ Router.map(function () {
   this.route('tasks');
   this.route('profile');
   this.route('identity');
+  this.route('page-not-found', { path: '/*' });
```
Comment (rohan09-raj) [inline app/router.js:16]: Can we name it `404` only ?

File: app/styles/app.css [hunk 2]
```diff
@@ -13,7 +13,7 @@
 @import 'new-signup.css';
 @import 'variables.css';
 @import 'user-status-modal.css';
-
+@import 'page-not-found.css';
 html,
```
Comment (rohan09-raj) [inline app/styles/app.css:17]: Please add a line space after the import

File: app/styles/page-not-found.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
+.not-found-container{
+  margin: auto;
+  text-align: center;
+}
+
+.not-found-img {
+  width: 70%;
```
Comment (rohan09-raj) [inline app/styles/page-not-found.css:7]: The image seems too big in the screenshot according to other elements. Can we please have an image of height around `400px` or less and accordingly the width of the image.
Also, ensure that it's responsive for all devices, text and the image

File: app/templates/page-not-found.hbs [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{{page-title 'Page Not Found'}}
+
+<div class="not-found-container">
+<img class="not-found-img" src="page-not-found.png" alt="page-not-found">
+<h1 class="not-found-text">The page you're looking for cannot be found!</h1>
+</div>
```
Comment (rohan09-raj) [inline app/templates/page-not-found.hbs:6]:
```suggestion
<div class="not-found">
  <img class="not-found__img" src="page-not-found.png" alt="page-not-found">
  <h1 class="not-found__text">The page you're looking for cannot be found!</h1>
</div>
```

File: app/styles/page-not-found.css [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+}
+
+.not-found-text{
+  font-size: 38px;
```
Comment (rohan09-raj) [inline app/styles/page-not-found.css:11]: I think default size of h1 would be good here

**Review by rohan09-raj — COMMENTED** [review]
Can we please have to screenshots of different screen sizes i.e. web, tablet  and mobile?

File: app/templates/404.hbs [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{{page-title 'Page Not Found'}}
+
+<div class="not-found-container">
+  <img class="not-found-img" src="page-not-found.png" alt="page-not-found">
+  <h1 class="not-found-text">The page you're looking for cannot be found!</h1>
```
Comment (rohan09-raj) [inline app/templates/404.hbs:5]: Please fix the class names as per the BEM convention

**Review by rohan09-raj — COMMENTED** [review]
.

File: app/styles/404.css [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+  text-align: center;
+}
+
+.not-found-img {
+  width: 60%;
+}
+
+.not-found-text{
+  font-weight: bold;
+}
```
Comment (rohan09-raj) [inline app/styles/404.css:12]: For the mobile devices, can we please make the font size a bit small somewhere around `18px` ?

Comment (satyam73) [reply app/styles/404.css:18]: > Image with media query added for small devices
image

**Review by rohan09-raj — APPROVED** [review]
Great work 🔥

**Pull request merged**